provider/kubernetes/kubernetes.go

```diff
@@ -186,6 +186,17 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				witelistSourceRangeAnnotation := i.Annotations[annotationKubernetesWhitelistSourceRange]
 				whitelistSourceRange := provider.SplitAndTrimString(witelistSourceRangeAnnotation)
 
+				priority := len(pa.Path)
+
+				priorityString, ok := i.Annotations[types.LabelFrontendPriority]
+				if ok {
+					priorityParsed, err := strconv.Atoi(priorityString)
+
+					if err == nil {
```

Comment:
We should not ignore errors.

When we encounter an error, we can either

- drop the associated frontend (by deleting it from the frontends map) or
- fall back to the default priority of 1.

We kind of do a mix in Traefik, so we cannot just point at any single pattern. Personally, I'm much more in favor of dropping the frontend as this makes the error explicit and way easier to identify. @dtomcej @errm thoughts?

In either case, the error should be logged.